templates/taxi/car_list.html

```diff
@@ -7,6 +8,11 @@ <h1>
       Create
     </a>
   </h1>
+  <form class="form-inline" action="" method="get">
+    {% csrf_token %}
+  {{ search_form|crispy }}
+  <input class="btn btn-secondary" type="submit" value="🔎">
+  </form>
```

Comment:
Please, move the search form to a separate file and reuse it where you need a search form